Services/Documents/DocumentLibraryService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Documents;
+
+namespace Andculture.Sitefinity.Testing.Services.Documents
+{
+    public class DocumentLibraryService : ODataService<DocumentLibrary>
+    {
+        public DocumentLibraryService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/documentlibraries";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.